sdk/servicebus/service-bus/package.json

```diff
@@ -111,7 +111,7 @@
     "@azure/core-client": "^1.0.0",
     "@azure/core-rest-pipeline": "^1.1.0",
     "@azure/core-tracing": "^1.0.0",
-    "@azure/core-paging": "^1.1.1",
+    "@azure/core-paging": "^1.3.1",
```

Comment:
nitpick: I would treat this as a new feature rather than a patch so perhaps the version can be bumped to 1.4.

Reply:
we can bundle this change with https://github.com/Azure/azure-sdk-for-js/pull/23404 in v1.4.0 so no need to update versions here.

Reply:
Yeah v1.4.0 is better.